### DISCUSSION TITLE
Looking to track changes if an event is published.

### DISCUSSION THREAD

**Discussion by chrisgrim** [owen-it/laravel-auditing#790] [Q&A] [answered]
| Q                 | A
| ----------------- | ---
| Bug?              | no
| New Feature?      | yes
| Framework         | Laravel
| Framework version | 7.25.0
| Package version   | ^10.0
| PHP version       | 7.4.7

Hi,
I am reaching out because I was hoping there was a way I could only track changes if my event is published. Currently it is tracking every change of the event. However I only want to track changes after the event has been published. I achieve this by changing my status column to 'p'.
Is there a way to say only track changes where status  == 'p' ?

**Comment by angeljqv** [comment, marked as answer]
I do this
```php
if ($myModel->status != 'p' ) { //not published
     MyModelClass::disableAuditing();
}

$myModel->save();
MyModelClass::enableAuditing();
```
